.github/workflows/publish-documentation.yml

```diff
@@ -18,29 +18,40 @@ jobs:
     steps:
       - name: Checkout source
         uses: actions/checkout@v3
+        with:
+          path: main
```

Comment:
main?

Reply:
arbitrary folder name because we need two different git repos downloaded.  i called one 'main' and one 'published'...

Reply:
Ah, dest directory, OK.